guides/common/modules/con_web-ui-overview.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Web-UI-Overview_{context}"]
+= {ProjectWebUI} Overview
+
+You can manage and monitor your {Project} infrastructure from a browser with the {ProjectWebUI}.
+
+In the {ProjectWebUI}, you can use the following navigation features:
+
+[cols="2,7", options="header"]
+|====
+| Navigation feature | Description
+| *Organization* dropdown | Choose the organization you want to manage.
+| *Location* dropdown | Choose the location you want to manage.
+| *Monitor* | Provides summary dashboards and reports.
+| *Content* | Provides content management tools.
+This includes content views, activation keys, and life cycle environments.
+| *Hosts* | Provides host inventory and provisioning configuration tools.
+| *Configure* | Provides general configuration tools and data, including host groups and Ansible content.
```

Comment:
I don't think it's a good idea to document all the navigation features. IMO it doesn't serve much purpose, and it would just get outdated again (the list in this PR is not new, I'm just [reusing a module](https://docs.theforeman.org/nightly/Administering_Project/index-foreman-el.html#Navigation_Tabs_in_the_Web_UI_admin) that's already in the Admin guide).

Instead, I rephrased the introduction to clarify that this is just an overview.